app/src/main/scala/org/alephium/explorer/RichAVector.scala

```diff
@@ -0,0 +1,30 @@
+// Copyright 2018 The Alephium Authors
+// This file is part of the alephium project.
+//
+// The library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the library. If not, see <http://www.gnu.org/licenses/>.
+
+package org.alephium.explorer
+
+import scala.collection.immutable.ArraySeq
+
+import org.alephium.util.AVector
+
+//TODO Add this to `org.alephium.util.AVector`
```

Comment:
We don't add this on purpose because this implicit conversion needs to allocate a new array. It's expensive for large vectors.

Reply:
I was not thinking of an implicit conversion, but just adding the `toArraySeq` function alongside [the existing toSeq](https://github.com/alephium/alephium/blob/d8b20ac2bb4578dc64aecb9e4d3a03063dd0c4b1/util/src/main/scala/org/alephium/util/AVector.scala#L650-L652)

Reply:
I see